### PR TITLE
show org info in `userdetails`

### PR DESCRIPTION
**Pull request by tonio**
ping @cmangeat

### CODE REVIEW

**Comment by tonio** [comment]
see #2125

**Comment by tonio** [comment]
@cmangeat on last thing, there a [3 strings](https://github.com/georchestra/georchestra/pull/2172/commits/5b9651734e2a424dd1e47079b9e006b6c8698707) that still needs to be translated. I put them into arbitrary tags, could you please plug with the translation system?

File: console/src/main/webapp/WEB-INF/i18n/application_fr.properties [hunk 1]
```diff
@@ -24,6 +24,9 @@ editUserDetailsForm.fieldset.userDetails=Détails du compte
 editUserDetailsForm.fieldset.credentials=Identification
 editUserDetailsForm.changePassword.link=Changer le mot de passe
 editUserDetailsForm.success=Compte actualisé
+editUserDetailsForm.organisation=Organisme
+editUserDetailsForm.areaOfCompetence=Aire de competence
```
Comment (fvanderbiest) [inline console/src/main/webapp/WEB-INF/i18n/application_fr.properties]: compétence

Comment (landryb) [reply console/src/main/webapp/WEB-INF/i18n/application_fr.properties]: 'Emprise de compétence' ? 'Aire' feels weird..

File: console/src/main/webapp/WEB-INF/i18n/application.properties [hunk 2]
```diff
@@ -24,6 +24,9 @@ editUserDetailsForm.fieldset.userDetails=User details
 editUserDetailsForm.fieldset.credentials=Credentials
 editUserDetailsForm.changePassword.link=Change Password
 editUserDetailsForm.success=User details updated successfully
+editUserDetailsForm.organisation=Organisation
```
Comment (fvanderbiest) [inline console/src/main/webapp/WEB-INF/i18n/application.properties]: Organization

Comment (tonio) [reply console/src/main/webapp/WEB-INF/i18n/application.properties]: Both are corrects. We use an «s» in the console.

**Comment by landryb** [comment]
Tested the tip of this branch, the map looks good ! Although it's still clickable, so users might think they can change their global area (which is of course forbidden) by selecting/unselecting areas, and then hitting 'send'. I think the map should really be readonly :)

I'm wondering if we should put a plaintext (paginated?) readonly list of org area similar to the one where you edit the area, for users to check their org has the correct list, or have an 'export area code list as csv' button reusing what's done in #2159 ?

Yes, it's a usecase we have...and when there's 200+ entries its painful to check by looking at a map... 

And still +1 for 'Emprise de compétence' which is a term i think everyone understands. At least that's the one we've been using here for years...

Visually, i really like the two pane approach. I'm wondering if we should move the 'Organisme' field from the left pane to the right pane as a single value line above 'Emprise de compétence', titling the right pane 'Organisme de rattachement' (so that the pane title has a static size, like 'Détails du compte' for the left pane)

This way, less text duplication.. right now on the page i have 'organisme' label & value twice.

**Comment by cmangeat** [comment]
A commercial could have said "territoire" for" zone de compétence", but maybe connotation are weird for a geographer...

**Comment by cmangeat** [comment]
I couldn't manage to reproduce /  hit the js assertion. Tried with an user with no org, tried with an user with an org, modifying or not the firstname.

About the post, there are two phases, first processing the input, registering the update, then providing an output, which reflect the changes. 9a4a080ff2b40a7fa5923f3aadb22f379cba5b4d deals with the second phase, i.e. the "refreshed" form has also to present the org.

**Pull request merged**